azerothcore/azerothcore-wotlk
# fix(scripts/Ulduar): Improve Hodir (#4703)

Continuing my abandoned PR

## Changes Proposed:

- Fix the helpers spawn before the encounter;
- Caches now spawn correctly before the encounter and not at the end;
- Fixed Toasty Fire Aura not preventing further Biting cold aura stacks;
- Removed hardcoded texts from Hodir (they were already on database);
- Break rare winter cache
- Hodir wearing weapon before encounter


## Tests Performed:
- Compiled and tested on Windows 10 with VSC19




## How to Test the Changes:

.modify hp 1000000
.cast 1908
.go c 136262

check the proposed changes and verify that they are all working

## Known Issues and TODO List:

- [x] Winter Cache GOBJECT is destroyed correctly without using NPCs around it


## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
